### PR TITLE
Converted flatten-nested-data (ksqlDB), geo-distance (ksqlDB), and hopping-windows (ksqlDB)

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

File: flatten-nested-data/ksql/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Transform nested JSON
```
Comment (davetroiano) [inline flatten-nested-data/ksql/README.md:1]:
```suggestion
# Transform nested JSON with ksqlDB
```

File: geo-distance/ksql/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Calculate lat-long
+
+You can use the ksqlDB [geo_distance]() function to calculate the distance between two latitude and longitude points.
```
Comment (davetroiano) [inline geo-distance/ksql/README.md:3]:
```suggestion
You can use the ksqlDB [`geo_distance`](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/scalar-functions/#geo_distance) function to calculate the distance between two latitude and longitude points.
```

File: hopping-windows/ksql/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Hopping windows in ksqlDB
+
+How to do hopping windows in ksqlDB.  A hopping window is a fixed-size window with an advance that is smaller than the window size. Due to that fact the advance is smaller than the window size, hopping windows contain overlapping results.
```
Comment (davetroiano) [inline hopping-windows/ksql/README.md:3]:
```suggestion
A hopping window is a fixed-size window with an advance that is smaller than the window size. Due to that fact the advance is smaller than the window size, hopping windows contain overlapping results.
```

File: hopping-windows/ksql/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Setup
+
+Imagine you have a topic of temperature readings, the first step is to create a stream over this topic:
```
Comment (davetroiano) [inline hopping-windows/ksql/README.md:7]:
```suggestion
Imagine you have a topic of temperature readings. The first step is to create a stream over this topic:
```

**Review by davetroiano — APPROVED** [review]
LGTM!

File: geo-distance/ksql/README.md (hunk 2 above)
Comment (bbejeck) [reply geo-distance/ksql/README.md:3]: whoops! looks like somebody got distracted!

**Review by davetroiano — APPROVED** [review]

**Pull request merged**